### PR TITLE
docs: add Portal Loop concept page

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR adds the Portal Loop concept page to the docs.
I propose we hold on merging this until `gno.land` is live.

EDIT: Portal Loop is live on [gno.land](https://gno.land) so we can merge this PR.

<details><summary>Contributors' checklist...</summary>

- [x] Added new tests, or not needed, or not feasible
- [x] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [x] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [x] Added references to related issues and PRs
- [x] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: docs/concepts/portal-loop.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
++----------------------------------+        |
+                 |                          |
+                 |                          |
+                 v                          |
++----------------------------------+        |
+|      Replay transaction data     |  ------+  
++----------------------------------+
+```
+
+The Portal Loop deployment can be found on [portal.gno.land:port](https://portal.gno.land).
```
Comment (thehowl) [inline docs/concepts/portal-loop.md:46]: Once it's in full steam, it should actually be on https://gno.land :)

Comment (waymobetta) [reply docs/concepts/portal-loop.md]: The portal loop will serve as official domain?

**Review by waymobetta — APPROVED** [review]

File: docs/concepts/portal-loop.md (hunk 1 above)
Comment (thehowl) [reply docs/concepts/portal-loop.md]: yes

Comment (moul) [reply docs/concepts/portal-loop.md]: The Portal Loop deployment can be found on [gno.land](https://gno.land).

Comment (leohhhn) [reply docs/concepts/portal-loop.md]: Fixed: [f36ab47](https://github.com/gnolang/gno/pull/1582/commits/f36ab47a39434f8f7d02c0d6ab7dc4f7e558bd92)

**Comment by leohhhn** [comment]
Should we add a note on decentralization? ie, Portal loop is, in the end, just a single node now, and is not really decentralized. For permanence & decentralization, use main.gno.land? 
Not sure if we will receive pushback because "its not decentralized".
@moul @thehowl

**Comment by thehowl** [comment]
> Not sure if we will receive pushback because "its not decentralized".

It's hard to make a "rolling testnet" like this decentralised. Not impossible, but it's already a feat as-is without considering multi nodes.

The portal loop is not a mainnet and we don't advertise it as such. It is a rolling testnet to develop Gno smart contracts that allows not falling behind on core developments as they are being made. `test4`, in this sense, is much closer to the notion of a testnet that can be decentralised. Likely going to be the devops focus after we finish working on the portal loop. So if somebody's annoyed at this, you can tell them to wait for test4.

---

More philosophically, if anybody complains that "it's not decentralised" my answer is you don't have to wait for the core team to get around to decentralising it. I know I'm a broken record on this but really open source projects have lived long past their original maintainers or companies supporting them simply by having people who cared, who took the original project source and improved it. So if you care about Gno and making its testnet decentralised, you study how you can decentralise it, propose a solution and try to do it. It's close to impossible that any reasonably made proposal for it gets seriously opposed. If you don't care enough to do all of that, then you can stay on Twitter.

This goes for 99% of "won't people complain" concerns. If they complain, tell them to get to code. If they don't know how to code, tell them to learn. Programming is the #1 topic with the most learning resources on the internet, and I learned nothing of what I know about programming through formal (uni) education or personal acquaintances.

**Comment by leohhhn** [comment]
@thehowl 

Fully agree with you, I just wanted to ask opinions on how to advertise it, since this docs page will be a source of truth, and we don't really advertise the Portal Loop anywhere else (except maybe the dev calls). This is why I think it is good to have a note on this within the page :)

I'm assuming most people won't even realize gno.land is actually the Portal Loop when it's live, unless they go to the docs and read about it.

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

**Pull request merged**